runlike/inspector.py

```diff
@@ -18,62 +17,83 @@ class Inspector(object):
 
     def __init__(self, container=None, no_name=None, pretty=None):
         self.container = container
+        self.image = None
         self.no_name = no_name
         self.output = ""
         self.pretty = pretty
-        self.facts = None
+        self.container_facts = None
+        self.image_facts = None
         self.options = []
 
     def inspect(self):
         try:
             output = check_output(
                 ["docker", "container", "inspect", self.container],
                 stderr=STDOUT)
-            self.facts = loads(output.decode())
+            self.container_facts = loads(output.decode())
+            self.image = self.get_container_fact("Config.Image")
+            output = check_output(
+                ["docker", "image", "inspect", self.image],
```

Comment:
I'll have to rewrite this to use "Image" instead of "Config.Image" here because if the user has pulled a new image with the same tag in the meantime, it will extract the default values from the new image version instead of the old one.